src/modules/create-wallet/ModuleCreateWalletSoftware.vue

```diff
@@ -139,12 +139,12 @@ export default {
      * @type - must be one of the this.types or an empty string (this will reset step to 0)
      */
     setType(newType) {
-      try {
```

Comment:
you can probably revert this once you merge the other PR with the Router config change.